### PR TITLE
Remove MetaMasterConfigClient initialization in the client

### PR DESCRIPTION
**Pull request by yyongycy**
### What changes are proposed in this pull request?

Remove MetaMasterConfigClient initialization in the client.

### Why are the changes needed?

Remove MetaMasterConfigClient initialization in the client.

### Does this PR introduce any user facing changes?

NA

### CODE REVIEW

**Comment by dbw9580** [comment]
Where should the clients get their config from instead, if this change is merged?

**Comment by yyongycy** [comment]
> Where should the clients get their config from instead, if this change is merged?

The client only reads locally so far

File: dora/tests/integration/src/test/java/alluxio/client/fs/PathConfigurationIntegrationTest.java [hunk 1]
```diff
@@ -1,119 +0,0 @@
...
-import org.junit.Assert;
-import org.junit.Before;
-import org.junit.Ignore;
-import org.junit.Rule;
-import org.junit.Test;
-
-/**
- * Integration tests for path level configurations.
- */
-public class PathConfigurationIntegrationTest {
```
Comment (dbw9580) [inline dora/tests/integration/src/test/java/alluxio/client/fs/PathConfigurationIntegrationTest.java:42]: why is this integration test removed?

Comment (yyongycy) [reply dora/tests/integration/src/test/java/alluxio/client/fs/PathConfigurationIntegrationTest.java:42]: PathConfig is already removed

**Review by dbw9580 — APPROVED** [review]

**Comment by yyongycy** [comment]
alluxio-bot, merge this please

**Pull request merged**